src/js/components/RegisterPage/SignUp.js

```diff
@@ -27,6 +32,17 @@ export default class Signup extends Component {
       this.state.password === this.state.confirmPassword
     );
   }
+  // (username, surname, passsword, email, phoneNumber, membershipType)
+  fun_register()
+  {
+    RegisterUser(this.state.name,
+      this.state.surname,
+      this.state.passsword,
+      this.state.email,
+      this.state.phoneNumber,
+      "free"
```

Comment:
what?

Reply:
Whats with this free?

Reply:
probably the membershipType. but this isnt how you set it bro

Reply:
Ambiguous feedback:

If you are referring to "free" parameter it is part of our database, as a default paramater that will be updated.

If your referring to the new lines I have fixed this to be on one line.